cypress/integration/e2e/link-checker/index.spec.ts

```diff
@@ -10,16 +10,7 @@
  */
 
 // FIXME: these are all broken links we should do something about
-const IGNORE_LIST = [
-  // Left these in there because they're being called from the sidebar nav.
-  // That will need to be refactored to pull from AirTable instead of packages.
-  'primitives/sibling-box',
-  'components/badge',
-  'components/display-pill-group',
-  'components/form-pill-group',
-  'patterns/data-export', // this randomly started failing and the page doesn't exist.
-  '/__/', // I don't know where this is being picked up
-];
+const IGNORE_LIST = [];
```

Comment:
Side note: would be good for us to get in the practice of including "add link to the ignore list" to ticket AC of all new components/packages, before the docs are written

Reply:
Maybe a better approach would be to fix the logic on the docsite that creates a broken link in the first place